### ISSUE TITLE
high contrast mode - info bar buttons should have borders

### ISSUE THREAD

**Issue by mousetraps** [Microsoft/vscode#16874]
Testing #15811

See below - can't tell they are buttons, color doesn't even change on hover. 
![image](https://cloud.githubusercontent.com/assets/762848/20992494/33fcdff2-bc99-11e6-8b32-96b0409759ff.png)

<hr>

- VSCode Version: Code - Insiders 1.8.0-insider (7499a9f3c565f4fbe5205e30c8664e5a5a11ae94, 2016-12-06T16:16:15.865Z)
- OS Version: Windows_NT ia32 10.0.14965
